### PR TITLE
Add conda install instructions to readme

### PR DESCRIPTION
**Pull request by brandon-b-miller**
Now that conda packages are available we can build out the install instructions here. 

https://anaconda.org/rapidsai-nightly/pynvjitlink

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,9 @@ for an editable install.
 
 ## Installation with Conda
 
-TODO
+```shell
+conda install rapidsai-nightly::pynvjitlink
```
Comment (bdice) [inline README.md:31]:
```suggestion
conda install -c rapidsai pynvjitlink
```

**Review by bdice — APPROVED** [review]

**Comment by bdice** [comment]
I re-pointed this to the stable release on `rapidsai` since our pip instructions use stable wheels. I'll merge this for now.

**Comment by jakirkham** [comment]
Thanks Brandon and Bradley! 🙏

**Pull request merged**